.github/workflows/hive.yaml

```diff
@@ -23,6 +23,8 @@ jobs:
         include:
           - simulation: rpc-compat
             run_command: just run-hive ethereum/rpc-compat "/eth_chainId|eth_getTransactionByBlockHashAndIndex|eth_getTransactionByBlockNumberAndIndex|eth_getCode|eth_getStorageAt|eth_call|eth_getTransactionByHash|eth_getBlockByHash|eth_getBlockByNumber|eth_createAccessList|eth_getBlockTransactionCountByNumber|eth_getBlockTransactionCountByHash|eth_getBlockReceipts|eth_getTransactionReceipt|eth_blobGasPrice|eth_blockNumber|ethGetTransactionCount"
+          - simulation: rpc-engine
```

Comment:
nit: we should add a `name` field and update the name of the job to use `${{ matrix.name }} instead of {{ matrix.simulaton }}`